api/config/config.go

```diff
@@ -213,7 +213,7 @@ type ImageBuilderConfig struct {
 	BuildTimeout                string           `validate:"required" default:"10m"`
 	KanikoImage                 string           `validate:"required" default:"gcr.io/kaniko-project/executor:v1.6.0"`
 	KanikoServiceAccount        string
-	Resources                   ResourceRequestsLimits `validate:"required"`
+	DefaultResource             ResourceRequestsLimits `validate:"required"`
```

Comment:
nit: Should we rename it to `DefaultResources` instead of `DefaultResource`? Because the struct contains two fields: resource requests and limits.